deployer.go

```diff
@@ -85,6 +89,7 @@ func (d *deployer) deployUnit(wantedUnit *schema.Unit) error {
 
 	wuf := schema.MapSchemaUnitOptionsToUnitFile(wantedUnit.Options)
 	cuf := schema.MapSchemaUnitOptionsToUnitFile(currentUnit.Options)
+	fmt.Printf("FOO:\n%T\n", wuf)
```

Comment:
fixed

On 9 June 2015 at 20:22, George Angel <notifications@github.com> wrote:

> In deployer.go
> <https://github.com/Financial-Times/coco-fleet-deployer/pull/1#discussion_r32052570>
> :
>
> > @@ -85,6 +89,7 @@ func (d *deployer) deployUnit(wantedUnit *schema.Unit) error {
> >
> >  	wuf := schema.MapSchemaUnitOptionsToUnitFile(wantedUnit.Options)
> >  	cuf := schema.MapSchemaUnitOptionsToUnitFile(currentUnit.Options)
> > +	fmt.Printf("FOO:\n%T\n", wuf)
>
> debug?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/coco-fleet-deployer/pull/1/files#r32052570>
> .
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.